torch/jit/_recursive.py

```diff
@@ -440,6 +441,8 @@ def init_fn(script_module):
         # 3. Copy @ignored/@unused methods and attrs from the original `nn_module` to the new ScriptModule.
         #    This ensures we can access these Python methods on the ScriptModule.
         for name in dir(nn_module):
+            if name in user_annotated_ignored_attributes:
```

Comment:
does this mean that the scriptmodule doesn't have the user annotated ignored attributes/methods anymore?